include/deal.II/multigrid/mg_coarse.h

```diff
@@ -28,6 +28,47 @@ DEAL_II_NAMESPACE_OPEN
 /*@{*/
 
 /**
+ * Coarse grid solver using preconditioners only. This is a little wrapper,
+ * transforming a preconditioner into
+ * a coarse grid solver.
+ *
+ * @author Denis Davydov, 2016.
+ */
+template<class VectorType = Vector<double> >
+class MGCoarseGridApplySmoother :  public MGCoarseGridBase<VectorType>
+{
+public:
+  /**
+   * Default constructor.
+   */
+  MGCoarseGridApplySmoother ();
+
+  /**
+   * Constructor. Store preconditioner for later use.
```

Comment:
Say "Store a pointer to the smoother for later use", to make clear that this object does not actually store the preconditioner and rather must be kept alive.